src/install_rules.ml

```diff
@@ -272,6 +272,7 @@ let symlink_installed_artifacts_to_build_install
     let dst =
       Path.append (Path.build install_dir)
```

Comment:
Not the fault of this PR, but this `dst` definition is super-ugly. I guess `relative_installed_path` should return a `Path.Relative.t` (or `Path.Local.t` in the new terminology that we're yet to adopt).

Reply:
Indeed that is the idea.